frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/radiogram-list/radiogram-card/radiogram-card-content/radiogram-card-content.component.html

```diff
@@ -1,4 +1,9 @@
 <ng-container [ngSwitch]="radiogram.type">
+    <app-radiogram-card-content-patient-count
+        *ngSwitchCase="'patientCountRadiogram'"
+        [radiogram]="$any(radiogram)"
```

Comment:
Afaik we chose to pass IDs instead of objects to new components.

Reply:
The existing content component and fallback components both received the object, maybe we should talk about this in general...

Reply:
We don't need to talk about this, we just need to refactor the other places. So it would be nice if we could not introduce more work for later. ^^

Reply:
This is true, I misread it in the fallback component. But I still do not understand the part about the content component. It is existent in the DOM and will be re-rendered if any changes to the radiogram happen which would cause the same effects onto the concrete content component. I do not see where my misunderstanding could lie

Reply:
I still dont understand the thing in the component but I changed it